test/model/adapters/sql/command_test.rb

```diff
@@ -0,0 +1,106 @@
+require 'test_helper'
+
+describe Lotus::Model::Adapters::Sql::Command do
+  let(:collection) { Object.new }
+  let(:query) do
+    query = Object.new
+    query.instance_variable_set(:@collection, collection)
+    query.define_singleton_method(:scoped, -> { @collection })
+    query
+  end
+
+  before do
+    @command = Lotus::Model::Adapters::Sql::Command.new(query)
+  end
+
+  describe '#create' do
+    it 'rescues database errors' do
```

Comment:
The spec description seems to be mismatching to the assertion. I think instead of testing the private method `rescue_database_error`, you should test it it with the public interface.